### PR TITLE
build: add imports for go generate tools

### PR DESCRIPTION
**Pull request by ucwong**

### CODE REVIEW

**Comment by fjl** [comment]
I've been thinking about this as well. This approach works, but has the problem that a request will be made to resolve the latest version of the tool every time `go generate` runs. Also, it is not possible to get deterministic output with this approach.

I think we should add a tools.go file in the project root that imports the tools. With this, it is possible to use `go run tool` (and not `go run tool@latest`) in the go:generate line, and the output will be deterministic.

**Review by fjl — APPROVED** [review]

**Comment by ucwong** [comment]
@fjl https://github.com/fjl/gencodec/pull/17 Check this PR

**Comment by fjl** [comment]
Done, and updated it here.

File: signer/fourbyte/4byte.go [hunk 1]
```diff
@@ -147000,6 +147000,9 @@ var _bindata = map[string]func() (*asset, error){
 	"4byte.json": _4byteJson,
 }
 
+// AssetDebug is true if the assets were built with the debug flag enabled.
+const AssetDebug = false
+
```
[the remark this answers is not in the document]
Comment (fjl) [reply signer/fourbyte/4byte.go:147005]: Ahh, I didn't see it correctly. This constant was added in go-bindata recently, and the output file was previously generated with an older version of go-bindata.

**Review by Kinou90100 — COMMENTED** [review]
#24682 
#24681 
#24680 
#24679 #24678 
#24677 #24676 
@Christoc90
@Kinou90100

**Review by mitumaruto — COMMENTED** [review]
#24790

**Pull request merged**